src/components/Modals/Onboarding.vue

```diff
@@ -409,13 +141,17 @@ export default {
           userData: {
             colleges: this.notPlaceholderOptions(this.displayOptions.college),
             majors: this.notPlaceholderOptions(this.displayOptions.major),
-            minors: this.notPlaceholderOptions(this.displayOptions.minor)
+            minors: this.notPlaceholderOptions(this.displayOptions.minor),
+            exam: this.notPlaceholderOptions2(this.displayOptions.exam),
```

Comment:
Should we name these better than 2 and 3?

Reply:
I'm not sure what you're referring to

Reply:
There are two methods here called 'notPlaceholderOptions' and 'notPlaceholderOptions2.' I think these names should be a bit more descriptive so we know what they mean and how they are different.

Reply:
Ohhhh they looked exactly the same to me I didn't notice LOL

Reply:
Haha exactly why we need to update it